### ISSUE TITLE
Displaying big tensors (?) : AttributeError: 'torch.FloatTensor' object has no attribute 'is_pointer'

### ISSUE THREAD

**Issue by LaRiffle** [OpenMined/Grid#190]
I was playing with the `Federated Learning Example.ipynb` and wanted to adapt the Boston/Denver example `DenverMLGridDemo.ipynb`. I had the following error that you can trigger by running the following:

`import torch`
`import torch.nn as nn`
`from grid.clients.torch import TorchClient`

`client = TorchClient(verbose=False)`

This works for many (i, j) small ex:
`nn.Parameter(torch.FloatTensor(4,3))`
But doesn't work for (i, j) "big" 
`nn.Parameter(torch.FloatTensor(26,39))`
`nn.Parameter(torch.FloatTensor(32,64))`
Throws
`AttributeError: 'torch.FloatTensor' object has no attribute 'is_pointer'`

**Comment by jvmancuso** [comment]
Yes, this is a known issue that affects PySyft as well.  I believe the root cause is in #171.  If I recall correctly, it's because we skip hooking into the `select` method in torch, which caused infinite recursions whenever we hooked in.  How I think we could go about this is to explicitly overwrite the __repr__ functions with our own versions that replace all torch-specific commands with their `old*` counterparts in TorchHook (so that when you call __repr__ on a local tensor, it calls pure torch code instead of hooked torch code.

**Comment by jvmancuso** [comment]
(Also, should mention that this would be a PR in PySyft, since that's where this code is going to live long-term).
